src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_QualByDepth.java

```diff
@@ -117,37 +117,43 @@
             return null;
         }
 
-        final List<Integer> standardDepth = getAlleleDepths(genotypes);
+        final List<Integer> standardDepth;
+        if (originalVC.hasAttribute(GATKVCFConstants.AS_VARIANT_DEPTH_KEY)) {
+            standardDepth = Arrays.stream(originalVC.getAttributeAsString(GATKVCFConstants.AS_VARIANT_DEPTH_KEY, "")
+                    .split(AnnotationUtils.ALLELE_SPECIFIC_SPLIT_REGEX)).mapToInt(Integer::parseInt).boxed().collect(Collectors.toList());
+        } else {
+            standardDepth = getAlleleDepths(genotypes);
+        }
         if (standardDepth == null) { //all no-calls and homRefs
             return null;
         }
 
-        //Parse the VC's allele-specific qual values
-        List<Object> alleleQualObjList = vc.getAttributeAsList(GATKVCFConstants.AS_QUAL_KEY);
-        if (alleleQualObjList.size() != vc.getNAlleles() -1) {
-            throw new IllegalStateException("Number of AS_QUAL values doesn't match the number of alternate alleles.");
-        }
-        List<Double> alleleQualList = new ArrayList<>();
-        for (final Object obj : alleleQualObjList) {
-            alleleQualList.add(Double.parseDouble(obj.toString()));
-        }
+        List<Integer> alleleQualList = parseQualList(vc);
+
 
         // Don't normalize indel length for AS_QD because it will only be called from GenotypeGVCFs, never UG
         List<Double> QDlist = new ArrayList<>();
         double refDepth = (double)standardDepth.get(0);
         for (int i = 0; i < alleleQualList.size(); i++) {
-            double AS_QD = -10.0 * alleleQualList.get(i) / ((double)standardDepth.get(i+1) + refDepth); //+1 to skip the reference field of the AD, add ref counts to each to match biallelic case
+            double AS_QD = alleleQualList.get(i) / ((double)standardDepth.get(i+1) + refDepth); //+1 to skip the reference field of the AD, add ref counts to each to match biallelic case
```

Comment:
Is it a breaking change that we're no longer multiplying by -10 here?

Reply:
No, I made the compensatory change for internal AS_QUAL to be Phred-scaled instead of log10.  It's a lot cleaner.